doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -850,6 +856,67 @@ storages. You can change the size of the allocated memory using the
 error can also be resolved by increasing the size of the physical
 cluster disk.
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+CrashLoopBackOff status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pods do not start and have the status ``CrashLoopBackOff``
+
```

Comment:
In short, this means that the container starts and crashes soon after due to an error in the code.